src/components/organisms/Chatbox/Chatbox.tsx

```diff
@@ -122,9 +122,9 @@ const Chatbox: React.FunctionComponent<PropsType> = ({
       setPrivateRecipient(undefined);
       if (!isInProfileModal && lastChat && lastChat.type === "private") {
         if (lastChat?.to === user.uid) {
-          setPrivateRecipient(userArray.find((u) => u.id === lastChat?.from));
+          setPrivateRecipient(worldUsers.find((u) => u.id === lastChat?.from));
         } else {
-          setPrivateRecipient(userArray.find((u) => u.id === lastChat?.to));
+          setPrivateRecipient(worldUsers.find((u) => u.id === lastChat?.to));
```

Comment:
Will this break private chat messages from users outside of `worldUsers`?

Reply:
No, since messages use different logic (chartUsers, etc)

Reply:
This appears to be directly related to private chat and is using `worldUsers` though? Can you please explain why you're confident this won't break things.

Reply:
TBH, I have no idea what this is chunk of code is about. The main private chat users' fetching happens in here
![image](https://user-images.githubusercontent.com/63194656/105774162-a76dd880-5f6d-11eb-9a20-387a69cb9394.png)

Here it decides if the chat is venue or private and fetches users accordingly.

I tried to debug this by console logging inside that useEffect, but I couldn't log anything no matter what I did in the UI. This chunk was written ~7 month ago. Is it possible that it's not functional anymore?

Reply:
It’s definitely possible that it’s not functional/needed anymore. I would need to look closer at the logic/detail around it to figure out if that’s the case or not though.

Perhaps @denisdimitrov has some insight/knowledge here?

Reply:
@mike-lvov RE: your comment in https://github.com/sparkletown/sparkle/pull/1047#discussion_r564088480

It looks like your screenshot is of `src/components/molecules/Chatbox/Chatbox.tsx`, whereas this comment thread was made about `src/components/organisms/Chatbox/Chatbox.tsx`. Were you aware of that?

Reply:
Given this `setPrivateRecipient` code appears to be in legacy code that will be deleted soon (see https://github.com/sparkletown/sparkle/pull/1047#discussion_r564959987), I don't think we need to be too concerned about fixing the issues I called out in this comment thread in this PR.